compiler/src/iree/compiler/Dialect/Stream/IR/StreamTypes.cpp

```diff
@@ -68,6 +68,8 @@ static llvm::cl::opt<bool> clResourceAliasMutableBindings(
     llvm::cl::desc(
         "Fuses bindings that are mutable instead of leaving them split."),
     llvm::cl::init(false));
+// TODO: Change this to discrete once task system scalability limits are
```

Comment:
if you do `// TODO(#15522): ...` then github/vscode will link it like cs/buganizer used to